### DISCUSSION TITLE
Is Ory for me?

### DISCUSSION THREAD

**Discussion by amks1** [ory/kratos#2075] [Q&A] [answered]
I just stumbled onto Ory. It looks like really great work, and I can see that this will shape up to be something of a gamechanger. From the docs I see that only the self-service function of Kratos is available right now. 

My use case would be to onboard an organization, and create all their users in bulk with predetermined credentials (only username and password, we don't want to enforce email addresses). Then I would need them to be able to SSO on all our apps (Flask, Django, Android Java, Node).. Would this be possible right now? I would much appreciate a confirmation, before I dive into the docs in detail. Thanks!

**Comment by vinckr** [comment, marked as answer]
Hello @amks1 
That is correct, our managed service `Ory Cloud` currently provides Ory Kratos APIs for authentication, autorization (Ory Keto) and federation (Ory Hydra) will be added soon. 

SSO is currently not a part of Ory Kratos at the moment - it is planned to be added soon.
Currently you can solve it by running Ory Hydra as OAuth2.0 provider, but SSO is not a use case best solved with OAuth2.0.

We do have SDKs for all popular languages.  

Don't hesitate to let me know if you have any more questions!

**Reply by vinckr** [reply]
Hey @amks1 ,
Just checking in if your question was answered or if there was still something unclear.
I will mark this as answered for now, but feel free to come back to the discussion any time.

Thanks a ton! 🐝
